samples/deployment/test-network/README.md

```diff
@@ -79,11 +79,12 @@ cd $FPC_PATH/samples/deployment/test-network
 ```
 
 Now we have the FPC Chaincode installed on the Fabric peers, but we still need to start our chaincode containers.
-Make sure you have set `TEST_CC_ID` to the same chaincode ID as used in the earlier step when building the chaincode.
+Make sure you have set `CC_ID` to the same chaincode ID as used in the earlier step when building the chaincode. Also confirm that `CC_PATH` is set to the location of the FPC chaincode code.
 
 ```bash
 # Start FPC container
-export TEST_CC_ID=echo
+export CC_ID=echo
+export CC_PATH=${FPC_PATH}/samples/chaincode/echo
```

Comment:
maybe the explicit export instruction in the beginning of this readme is enough and we can save those two lines here. The ext above also tells the user to check that `CC_ID` and `CC_PATH` is set accordingly.

Reply:
Yeah, I also thought of that ...

Reply:
Done!